types/validator_set_test.go

```diff
@@ -1260,6 +1246,75 @@ func verifyValSetUpdatePriorityOrder(t *testing.T, valSet *ValidatorSet, cfg tes
 	}
 }
 
+func TestValSetUpdateOverflowRelated(t *testing.T) {
+	// short name for test readability
+	big := MaxTotalVotingPower
```

Comment:
why not use MaxTotalVotingPower directly? I think big obscures the meaning and can potentially bring more confusion rather than clarity

Reply:
I found it easier to read/ write the tests.

Reply:
Maybe call it max then? Why big?

Reply:
Because it doesn't have to be the max really, any value close enough to max would cause a problem in the earlier versions. Anyways, I changed it to `MaxTotalVotingPower` as per your earlier comment.